src/test/java/io/strimzi/kafka/bridge/http/services/SeekService.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.http.services;
+
+import io.strimzi.kafka.bridge.BridgeContentType;
+import io.strimzi.kafka.bridge.utils.Urls;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.client.HttpRequest;
+import io.vertx.ext.web.client.WebClient;
+import io.vertx.ext.web.codec.BodyCodec;
+
+import static com.google.common.net.HttpHeaders.CONTENT_LENGTH;
+import static com.google.common.net.HttpHeaders.CONTENT_TYPE;
+
+public class SeekService extends BaseService {
+
+    public SeekService(WebClient webClient) {
+        super(webClient);
+    }
+
+    // Seek basic requests
+
+    public HttpRequest<JsonObject> positionsRequest(String groupId, String name, JsonObject json) {
+        return postRequest(Urls.consumerInstancesPositions(groupId, name))
+                .putHeader(CONTENT_LENGTH, String.valueOf(json.toBuffer().length()))
+                .putHeader(CONTENT_TYPE, BridgeContentType.KAFKA_JSON)
+                .as(BodyCodec.jsonObject());
+    }
+
+    public HttpRequest<JsonObject> positionsBeginningRequest(String groupId, String name, JsonObject json) {
+        return postRequest(Urls.consumerInstancesPositionsBeginning(groupId, name))
+                .putHeader(CONTENT_LENGTH, String.valueOf(json.toBuffer().length()))
+                .putHeader(CONTENT_TYPE, BridgeContentType.KAFKA_JSON)
+                .as(BodyCodec.jsonObject());
+    }
+
+    public HttpRequest<JsonObject> positionsBeginningEnd(String groupId, String name, JsonObject json) {
+        return postRequest(Urls.consumerInstancesPositionsEnd(groupId, name))
+                .putHeader(CONTENT_LENGTH, String.valueOf(json.toBuffer().length()))
+                .putHeader(CONTENT_TYPE, BridgeContentType.KAFKA_JSON)
+                .as(BodyCodec.jsonObject());
+    }
```

Comment:
these three methods are pretty the same. The only different thing is the URL. I guess you can factor out a common method to be called by them.